vc_vidyo/indico_vc_vidyo/api/client.py

```diff
@@ -79,9 +79,28 @@ def create_room_object(self, **kwargs):
     def find_room(self, extension):
         filter_ = self.client.factory.create('Filter')
         filter_.query = extension
+        filter_.limit = 40
         filter_.dir = 'DESC'
 
-        return self.soap.getRooms(filter_).room
+        target_room = None
+        counter = 0
+        from indico_vc_vidyo.plugin import VidyoPlugin
+        while True:
+            filter_.start = (counter*filter_.limit)
```

Comment:
Missing whitespace around the `*` operator as per [PEP8](https://www.python.org/dev/peps/pep-0008/).